.local/bin/workman

```diff
@@ -58,6 +58,13 @@ if [ ! -f ./${WORKMAN_CONFIG_FILE} ]; then
     echo "BUILD_COMMAND=\"python2 x.py build -j8\""
     echo "# Command to run after creating a new working directory."
     echo "AFTER_COMMAND=\"rustup toolchain link \$(basename \$(pwd))-stage1 ./build/x86_64-unknown-linux-gnu/stage1\""
+    echo
+    echo "# Command to run in order to continue working in a given working directory"
+    echo "WORK_ATTACH_COMMAND=\"tmux new -A -s \\\"\\\${PROJECT_NAME}/\\\${TASK}\\\"\""
```

Comment:
Perhaps `SESSION` instead of `WORK`?

```suggestion
    echo "SESSION_REATTACH_COMMAND=\"tmux new -A -s \\\"\\\${PROJECT_NAME}/\\\${TASK}\\\"\""
```

Reply:
Could you also add these as defaults above where we source `.workman_config` so that existing config files don't need to be updated with these variables?

Reply:
Hm. I don't know how to do that, I'll look into that

Reply:
I think if you just added lines like below to the top of the file (~L23) then if the user didn't specify them in `.workman_config` their values wouldn't be replaced thus making sure they're always defined for existing configuration files that won't have declared them:

```bash
WORK_ATTACH_COMMAND="tmux new -A -s \"\${PROJECT_NAME}/\${TASK}\""
```